Update radical-stroke-related data files

tonton-pixel opened the pull request:
Updated versions of the `aj17-kanji.txt` and `aj17-radical-stroke.pdf` files provided by Dr Ken Lunde, reflecting the corrections made to the `kRSAdobe_Japan1_6` field of the Unihan database, planned for Unicode Version 14.0 next year.

> U+5F17	kRSAdobe_Japan1_6	C+3574+57.3.2
> U+5F18	kRSAdobe_Japan1_6	C+1986+28.2.3 C+1986+57.3.2
> U+5FE9	kRSAdobe_Japan1_6	C+14533+61.4.4
> U+6E20	kRSAdobe_Japan1_6	C+1678+75.4.8 C+1678+85.3.9
> U+72AE	kRSAdobe_Japan1_6	C+18056+94.4.1
> U+79C1	kRSAdobe_Japan1_6	C+2226+28.2.5 C+2226+115.5.2
> U+7C9B	kRSAdobe_Japan1_6	C+2391+119.6.5 C+2391+129.5.6
> U+FA3D	kRSAdobe_Japan1_6	C+13326+61.3.7
> U+FA4A	kRSAdobe_Japan1_6	C+7732+96.4.8

tonton-pixel committed:
Add files via upload

Added updated files provided by Dr Ken Lunde

hatchzo commented:
I think the delimit is wrong in the line :
 1986	25-16	25-16	25-16	25-16	25-16	*	*	*	Jinmei	*	*	*	*	*	*	*	U+5F18	U+5F18	<U+5F18,U+E0100>	57,3,2:28.2.3

It should be :
 1986	25-16	25-16	25-16	25-16	25-16	*	*	*	Jinmei	*	*	*	*	*	*	*	U+5F18	U+5F18	<U+5F18,U+E0100>	57,3,2:28,2,3

Could you check that?

tonton-pixel committed:
Add files via upload

Added revised file aj17-kanji.txt provided by Dr Ken Lunde, correcting the delimiter issue

tonton-pixel commented:
Thanks for catching the delimiter issue. I added a second commit with the revised file aj17-kanji.txt provided by Dr Ken Lunde. The PDF doesn't have this issue.
